core-developers/experts.rst

```diff
@@ -355,6 +357,7 @@ release management  tarekziade, malemburg, benjaminp, warsaw,
                     gvanrossum, anthonybaxter^, merwok, ned-deily,
                     birkenfeld, JulienPalard
 str.format          ericvsmith*
+subinterpreters     kumaraditya303
```

Comment:
Here you should definitely add Eric Snow

Reply:
Added